maintenance/wikia/updateDatawarePages.php

```diff
@@ -207,7 +206,6 @@ private function updateDatawarePage( $dbw, $localPage, $datawarePage ) {
 				'page_wikia_id' => $wgCityId,
 				'page_id' => $localPage->page_id,
 				'page_namespace' => $localPage->page_namespace,
-				'page_title_lower' => mb_strtolower( $localPage->page_title ),
```

Comment:
We may get rid of this script completely (being considered in https://wikia-inc.atlassian.net/browse/SUS-1789).